Lib/turtledemo/__main__.py

```diff
@@ -171,15 +170,23 @@ def __init__(self, filename=None):
         self.output_lbl = Label(root, height= 1, text=" --- ", bg="#ddf",
                                 font=("Arial", 16, 'normal'), borderwidth=2,
                                 relief=RIDGE)
-        self.start_btn = Button(root, text=" START ", font=btnfont,
-                                fg="white", disabledforeground = "#fed",
-                                command=self.startDemo)
-        self.stop_btn = Button(root, text=" STOP ", font=btnfont,
-                               fg="white", disabledforeground = "#fed",
-                               command=self.stopIt)
-        self.clear_btn = Button(root, text=" CLEAR ", font=btnfont,
-                                fg="white", disabledforeground="#fed",
-                                command = self.clearCanvas)
+        if darwin:  # Leave Mac button colors alone - #44254.
```

Comment:
It duplicates a lot of code which is the same on all platforms. Would not be better to factor out variable parts?
```
if darwin:
    foreground = ...
    disabledforeground = ...
else:
    foreground = ...
    disabledforeground = ...
```

Reply:
This is an interim patch changing Mac only that I hope gets into .0b2 to workaround the tk bug on Mac (see issue).  I initially removed both colors for Mac, but the default buttons are buggy depending on the users choice of theme.  The default disabled foreground on Mac is fine as far as I know, and I don't especially care as being readable then does not matter.  After I decide what to do with non-Mac, I will consider whether to even set disabledforeground on non-Mac.   I want to try multiple colors on non-Mac, and if so, the code will not be exactly parallel.  And maybe use ttk widgets.